src/redux/sagas/wallet/onboard.ts

```diff
@@ -21,15 +22,18 @@ const { formatMessage } = intl({
   'info.text': 'Connect your wallet to log in',
 });
 
+const ganacheAccountsUrl = new URL(
+  process.env.GANACHE_ACCOUNTS_ENDPOINT || 'http://localhost:3006',
+);
+
 const getDevelopmentWallets = async () => {
   // variable injected by webpack
   // @ts-ignore
   // if we're using the webpack.dev config, include dev wallets
   if (!WEBPACK_IS_PRODUCTION) {
-    const { private_keys: ganachePrivateKeys } = await import(
-      // @ts-ignore
-      '../../../../amplify/mock-data/colonyNetworkArtifacts/ganache-accounts.json'
-    );
+    const { private_keys: ganachePrivateKeys } = (
+      await axios.get(`${ganacheAccountsUrl.href}ganache-accounts.json`)
```

Comment:
I _think_ we want to get rid of axios eventually as it's not really needed. For this simple stuff, maybe you could use the `fetch` API?